style/golf.mss

```diff
@@ -0,0 +1,66 @@
+#landcover[zoom >= 14] {
+  ::high-zoom[zoom >= 12] {
+    [feature = 'golf_green'],
+    [feature = 'golf_tee'] {
+      polygon-fill: mix(@grass, @park, 50%);
+    }
+    [feature = 'golf_fairway'],
+    [feature = 'golf_driving_range'] {
+      polygon-fill: @grass;
+    }
+    [feature = 'golf_bunker'] {
+      polygon-fill: @sand;
+      polygon-pattern-file: url('symbols/beach.png');
+    }
+    [feature = 'golf_rough'] {
+      polygon-fill: mix(@scrub, @grass, 50%);
```

Comment:
I have replaced this `mix()` with `@grass`, overlaid with the `golf_rough.svg` pattern from the ac-style. The result is a subtle differentiation rather than a large color difference.